client/components/QuestionList.jsx

```diff
@@ -34,12 +33,12 @@ const QuestionList = ({ questionList, answerQuestion }) => {
               className="headUnit"
             >
               <div className="grid-container">
-                <p className="author">{question.questionAuthor}</p>
-                <p className="createdAt"><Moment fromNow>{question.questionCreatedAt}</Moment></p>
+                <p className="author">{question.questionauthor}</p>
+                <p className="createdAt"><Moment fromNow>{question.questioncreatedat}</Moment></p>
```

Comment:
it is really these minor differences that can create such big problems! nice fixes :)